docs/source/rst_quick_ref.rst

```diff
@@ -0,0 +1,449 @@
+
+  .. |copy| unicode:: 0xA9 .. copyright sign
+
+Directives
+~~~~~~~~~~
+
+line-block
+++++++++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. line-block::
+            Subsequent indented lines comprise
+            the body of the sidebar, and are
+            interpreted as body elements.
+    
+     - .. line-block::
+            Subsequent indented lines comprise
+            the body of the sidebar, and are
+            interpreted as body elements.
+   * - :: 
+
+        | But can a bee be said to be
+        |     or not to be an entire bee,
+        |         when half the bee is not a bee,
+        |             due to some ancient injury?
+     -
+        | But can a bee be said to be
+        |     or not to be an entire bee,
+        |         when half the bee is not a bee,
+        |             due to some ancient injury?
+
+code
+++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+
+   * - ::
+    
+        .. code:: python
+
+            def hey(**kargs):
+                '''
+                Do something.
+                :Parameters:
+                    size
+                        The size of the fox (in meters)
+                :rtype: str
+                :return: The number of foxes
+                '''
+                pass
+
+     - .. code:: python
+
+        def hey(**kargs):
+            '''
+            Do something.
+            :Parameters:
+                size
+                    The size of the fox (in meters)
+            :rtype: str
+            :return: The number of foxes
+            '''
+            pass
+
+   * - ::
+    
+        .. python:: 
+
+            def hey(**kargs):
+                '''
+                Do something.
+                :Parameters:
+                    size
+                        The size of the fox (in meters)
+                :rtype: str
+                :return: The number of foxes
+                '''
+                pass
+
+     - .. python:: 
+
+        def hey(**kargs):
+            '''
+            Do something.
+            :Parameters:
+                size
+                    The size of the fox (in meters)
+            :rtype: str
+            :return: The number of foxes
+            '''
+            pass
+
+math
+++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. math::
+
+        α_t(i) = P(O_1, O_2, … O_t, q_t = S_i λ)
+
+     - .. math::
+
+        α_t(i) = P(O_1, O_2, … O_t, q_t = S_i λ)
+
+raw
++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. raw:: html
+
+            <hr />
+
+     - .. raw:: html
+
+            <hr />
+
+figure & image
+++++++++++++++
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Plain text
+     - Result
+     
+   * - :: 
+
+        .. figure:: https://khms1.googleapis.com/kh?v=878&hl=en-US&x=2273&y=3006&z=13
```

Comment:
Could we use a local file instead? Unfortunately pydoctor doesn't have a logo, but perhaps we can use the Twisted or Python logo, those should be pretty small.